capstone/src/main/webapp/js/maps.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Displays the map onto the dashboard with hardcoded values.
+ * @return {void}
```

Comment:
I believe the `@return` tag is only needed if the function actually returns a parameter. If you want you can keep it though.